infra/modules/gcp-psoxy-rest/variables.tf

```diff
@@ -50,20 +50,6 @@ variable "path_to_config" {
   default     = null
 }
 
-variable "salt_secret_id" {
-  type        = string
-  description = "Id of the secret used to salt pseudonyms"
-}
-
-variable "salt_secret_version_number" {
-  type        = string
-  description = "Version number of the secret used to salt pseudonyms"
-  validation {
-    condition     = can(regex("^([0-9]+)|latest$", var.salt_secret_version_number))
-    error_message = "Version number must be a number or 'latest'."
-  }
-}
-
```

Comment:
so this is breaking change, right? for anyone invoking `gcp-psoxy-rest` module directly, they'll need to stop passing these, right?

if so, 1) add to `CHANGELOG` and 2) possibly leave these in `variables.tf`, just mark as deprecated?